lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -56,12 +64,30 @@ public function init() {
 	}
 
 	/**
-	 * Get the list of fonts.
+	 * Get the list of registered fonts.
+	 *
+	 * @return array
```

Comment:
```suggestion
	 * @return array[]
```
When populated this returns an array of arrays.